src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipService.java

```diff
@@ -102,4 +105,10 @@ private boolean hasExistingLimitedPartnershipSubmission(Transaction transaction)
         }
         return false;
     }
+
+    void updateLimitedPartnershipSubmissionWithSelfLink(LimitedPartnershipSubmissionDao submission,
```

Comment:
This method should remain `private` as only called from within this class. The unit-testing has to be done through the public `createLimitedPartnership()` method.

Reply:
method made private testing through createLimitedPartnership() method